data/_ui/tooltips.txt

```diff
@@ -1320,6 +1320,9 @@ tip "Screen mode"
 tip "VSync"
 	`Toggle vsync between off, on, and adaptive (should your computer support it).`
 
+tip "Camera acceleration"
+	`Toggle whether the flagship is fixed in the center of the screen, or moving around slightly when accelerating.`
```

Comment:
```suggestion
	`Toggle whether the camera is fixed on the flagship, or begins to offset from the flagship while accelerating.`
```
This seems to be more direct and clear about _camera_ acceleration?

Or if you want to keep it about the flagship instead of the camera:
```suggestion
	`Toggle whether the flagship is fixed in the center of the screen, or begins to offset from the center when accelerating.`
```